src/feedback/components/AmountPicker.vue

```diff
@@ -47,25 +47,33 @@
         <QSlider
           :value="limitedValue"
           :min="0"
-          :max="70"
+          :max="100"
           :step="0.5"
           label
-          class="self-center q-mx-md"
+          class="self-center q-mr-lg"
+          style="margin-left: 8px"
```

Comment:
Could we use a `q-ml-?` classname here? (or maybe `q-my-lg` if they can be the same (or do I mean `q-mx-lg` I get confused between x and y))